scss/_utilities/_clearfix.scss

```diff
@@ -0,0 +1,21 @@
+// ==========================================================================
+// Clearfix
+//
+// You can either use the mixin or @extend .clear.
```

Comment:
I've used neither syntax before, so this doesn't help me understand how to use this. And the `@extend .clear` looks odd, since only `.clearfix` is defined below.